dill/_dill.py

```diff
@@ -2117,24 +1868,21 @@ def save_property(pickler, obj):
 @register(classmethod)
 def save_classmethod(pickler, obj):
     log.info("Cm: %s" % obj)
-    im_func = '__func__' if PY3 else 'im_func'
+    im_func = '__func__'
```

Comment:
could just use `__func__` in `getattr` directly below.

Reply:
Missed this one.